README.md

```diff
@@ -6,6 +6,23 @@ This experiment creates and trains three neural networks, termed Alice, Bob, and
 
 This repository was developed as part of the [IBM Code Model Asset Exchange](https://developer.ibm.com/code/exchanges/models/)
 
+## Model Metadata
+| Domain | Application | Industry  | Framework | Training Data | Input Data Format |
+| ------------- | --------  | -------- | --------- | --------- | -------------- |
+| Security | Cryptography | General | TensorFlow | Randomly generated text | Text |
+
+## References
+
+* Abadi & Andersen, [Learning to Protect Communications with Adversarial Neural Cryptography](https://arxiv.org/abs/1610.06918), ,2016.
+* [TensorFlow implementation](https://github.com/tensorflow/models/tree/master/research/adversarial_crypto)
+
+## Licenses
+
+| Component | License | Link  |
+| ------------- | --------  | -------- |
+| This repository | [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0) | [LICENSE](LICENSE) |
+| Model Code (3rd party) | [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0) | [TensorFlow Models](https://github.com/tensorflow/models/blob/master/LICENSE)|
+
 # Quickstart
 
 ## Prerequisites
```

Comment:
While we are making this change I think we should add the detail about "if you don't have an instance of WML" with the IBM Cloud linke